calico_containers/pycalico/netns.py

```diff
@@ -180,6 +179,28 @@ def remove_ip_from_ns_veth(container_pid, ip, veth_name_ns):
                        "dev", "%(device)s" % veth_name_ns])
 
 
+def provision_veth(self, endpoint, pid, ip, veth_name_ns):
+    """
+    Create the veth, move into the container namespace, add the IP and
+    set up the default routes.
+
+    :param endpoint: The endpoint object to provision the veth on
+    :param pid: The PID of the namespace to operate in
+    :param ip: The IPAddress to provision.
+    :param veth_name_ns: The name of the interface in the namespace
+    :return The mac address of the veth as a string
+    """
+    create_veth(endpoint.name, endpoint.temp_interface_name)
+    move_veth_into_ns(pid, endpoint.temp_interface_name, veth_name_ns)
+    add_ip_to_ns_veth(pid, ip, veth_name_ns)
```

Comment:
We should be getting the IPs that have been saved into the endpoint, and applying those to the veth. Change this to:
```
for ip in ep.ipv4_nets:
    add_ip_to_ns_veth(pid, ip, veth_name_ns)
```